apps/token-manager/app/src/script.js

```diff
@@ -232,9 +220,6 @@ function loadTokenSettings(token) {
 
 function loadTokenSymbol(token) {
```

Comment:
And similar to the other comments, we could probably remove `loadTokenSymbol()`.